info.py

```diff
@@ -8,7 +8,9 @@
 
 
 authors = ("@classabbyamp", "@0x5c.io")
-description = """A bot with various useful ham radio-related functions, written in Python."""
+description = """A bot with various useful ham radio-related functions, written in Python.
+
+qrm is a <:miaowware_irl:678275824436379678> MiaowWare project."""
```

Comment:
Does this line still work with prepended whitespace to make it align with the first one? Might be somewhat more readable with the empty line replaced by `\n` too?